@stellar/design-system/package.json

```diff
@@ -19,7 +19,10 @@
     "clean": "rimraf ./build",
     "start": "run-p clean watch:tsc watch:dev"
   },
-  "peerDependencies": {},
+  "peerDependencies": {
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2"
+  },
```

Comment:
Useful link for why these should be peer deps: https://classic.yarnpkg.com/en/docs/dependency-types/#toc-peerdependencies